contributed_definitions/nyaml/NXlens_em.yml

```diff
@@ -1,36 +0,0 @@
-category: base
```

Comment:
Can confirm Rubel's observation NXlens_em files appeared twice one with yml one with yaml ending
Surplus different fields in them. Manually I compared yesterday the content of both and integrated the correct content in NXlens_em.yaml